inst/@sym/sym.m

```diff
@@ -277,7 +277,6 @@
   end
 
   asm = {};
-  check = true;
```

Comment:
I was trying to avoid the most possible call python_cmd every time, the main reason is https://github.com/cbm755/octsympy/pull/561, to can use classdef we will run an extra lines script every time we call that function for the limitations, so we can avoid write a lot of code and reduce it....

If you want remove it maybe instead keep it as comments? Just to don't loose the idea.